pkg/network/iface/bond.go

```diff
@@ -165,11 +165,34 @@ func getSlaves(index int) ([]netlink.Link, error) {
 	return links, nil
 }
 
-func compareBond(b1, b2 *netlink.Bond) bool {
-	if b1.Name == b2.Name && b1.MTU == b2.MTU && b1.Mode == b2.Mode &&
-		b1.HardwareAddr.String() == b2.HardwareAddr.String() && b1.TxQLen == b2.TxQLen {
-		return true
+func compareBond(old, new *netlink.Bond) bool {
```

Comment:
It would be nice to have a unit test for this function

Reply:
The reason why we need to ignore the default value depend on the package [vishvananda/netlink](https://github.com/vishvananda/netlink). The function is very easy which is the reason why there isn't a unit test.

Reply:
Adding more annotations to explain why it's necessary to ignore the default value would be more helpful.